platformio_tubes.ini

```diff
@@ -14,14 +13,13 @@ build_flags = -O2
   -D WLED_DISABLE_HUESYNC
   -D WLED_DISABLE_WEBSOCKETS
   -D WLED_DISABLE_ADALIGHT
+  -D WLED_DISABLE_ESPNOW
```

Comment:
several of the boards don't have a mic, so I've disabled it by default.  It is added back in dig2go which has a mic

Reply:
as for the WLED_DISABLE_ESPNOW, in 0.14 it just enables and ESPNOW remote control when not defined.  Do we need that for any tubes build?  I disabled it to avoid any conflict with the networking.  Perhaps it could be re-enabled, but what value does it add?

Reply:
This adds the support for the mic back in by adding the AR libs

```
[tubes]
extends = tubes_no_mic
build_flags = ${tubes_no_mic.build_flags}
  ${esp32.AR_build_flags}  
lib_deps = ${tubes_no_mic.lib_deps}
  ${esp32.AR_lib_deps}

Reply:
yep as in our conversation - I'm just noting that ESPNow (used to sync) doesn't seem to have anything to do with the mic or not-mic state of the board. So it seems odd to turn it off in a profile that is no-mic.

We don't use DISABLE_ESPNOW in our branch, but I can imagine using common code in the future where we just hook into the core ESPNow code. I don't really care one way or the other until then